database/things/amazon-dynamodb.pldb

```diff
@@ -0,0 +1,5 @@
+title Amazon DynamoDB
+appeared 2012
+type Proprietary NoSQL cloud-based database management system
```

Comment:
this would be `description`.

type would be `database`.

or to talk about the language to query the database type would be `queryLanguage`.

Reply:
got it, will update! 🔨

Reply:
done 🔨